jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-api/src/main/java/org/jbpm/workbench/cm/service/CaseManagementService.java

```diff
@@ -69,8 +70,18 @@
 
     void addDynamicUserTaskToStage(String containerId, String caseId, String stageId, String name, String description, String actors, String groups, Map<String, Object> data);
 
+    void addDynamicTask(String containerId, String caseId, String nodeType, String name, Map<String, Object> data);
+
+    void addDynamicTaskToStage(String containerId, String caseId, String stageId, String nodeType, String name, Map<String, Object> data);
```

Comment:
Just for consistency reasons, do we want to keep methods which handle dynamic service tasks in the interface while completely removing dynamic service task from CaseActionType enum?